feature(base-service): change typdef of dataId parameter in caching methods (#831)
plentymarkets/terra-components: CHANGELOG.md

###Unreleased

> * **terra-base.service** change key param typedef from caching methods

Reviewer: ```suggestion
### Features
* **terra-base.service** change typdef of dataId parameter in caching methods
```

<a name="3.5.0"></a>

> # 3.5.0 (XX.XX.XXXX)

Reviewer: 3.5.1 wurde bereits released
